README.md

```diff
@@ -37,7 +37,7 @@ distributions from the [Releases](https://github.com/terraform-providers/terrafo
 Download and uncompress the latest release for your OS. This example uses the linux binary.
 
 ```sh
-> wget https://github.com/terraform-providers/terraform-provider-helm/releases/download/v0.7.0/terraform-provider-helm_v0.7.0_linux_amd64.tar.gz
```

Comment:
This is out of the scope of this change

Reply:
sure, I will submit a separate PR to fix this release reference issue as well. 

Let me know if that works.